src/invalid-json-stringify.ts

```diff
@@ -0,0 +1,84 @@
+// invalid-json-stringify.ts
+
+import type { Rule } from 'eslint';
+import getDocumentationUrl from './get-documentation-url';
+
+/*
+ * Copyright (c) 2021-2024 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+export const ruleId = 'invalid-json-stringify';
```

Comment:
not necessarily against it, but what was the thinking behind making this a `const`?

Reply:
good question, first of all the rule identifier normally is repeated in many occations like implementation, tests, configurations etc.
secondly there seems to be potential to centralize the managment for certain aspects (e.g. document url) so i thought each rule module can export a rule id plus the essential meta/implementation and let `index.ts` handle those stuff.